samples/react-native/webrequest/src/App.fs

```diff
@@ -5,6 +5,9 @@ open Fable.Elmish
 open Fable.Import.Fetch
 open Fable.Helpers.Fetch
 open Fable.Core.Extensions
+open Fable.Extras.Promise
```

Comment:
Just opening `Fable.Extras` is good enough. I should actually mark `Cmd` module us unopable, because I want no ambiguity about it...